<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Potvrď</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_geog">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>87</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>64</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Kód studenta</string>
   </property>
   <property name="buddy">
    <cstring>lineEdit</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>9</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>35</y>
     <width>77</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Jméno studenta</string>
   </property>
   <property name="buddy">
    <cstring>lineEdit_2</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>35</y>
     <width>133</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>61</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Body historie</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_hist">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>61</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>87</y>
     <width>65</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Body zeměpis</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>34</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Celkem</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>50</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Úspěšnost</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_perc">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>150</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>spinBox_hist</tabstop>
  <tabstop>spinBox_geog</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>pressed()</signal>
   <receiver>Dialog</receiver>
   <slot>disp_message()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>disp_message()</slot>
 </slots>
</ui>
